### DISCUSSION TITLE
Why do we have our own Metrics code?

### DISCUSSION THREAD

**Discussion by lavocatt** [artemiscloud/activemq-artemis-self-provisioning-plugin#232] [General]
I'm asking this because the console seems to already have a bunch of code dedicated to plotting performances:
![image](https://github.com/artemiscloud/activemq-artemis-self-provisioning-plugin/assets/86971992/b970129a-f99d-4a0c-aa57-4cfbbd679ae9)

Since there's also more types of graphs our custom code is handling, this begs the question about why should we keep the custom metrics?

**Comment by RoddieKieley** [comment]
The idea here was to do what was easily available first during the early days of the project with the idea that in the future we were going to need much more in depth charting capabilities of data not just from the prometheus end point on the broker but for data through jolokia as well that wouldn't be covered here.

**Reply by lavocatt** [reply]
Thanks!
